src/components/PopupWithForm.js

```diff
@@ -21,7 +24,17 @@ export default class PopupWithForm extends Popup {
     this._popupForm.addEventListener("submit", (e) => {
       e.preventDefault();
       this._handleFormSubmit(this._getInputValues());
-      this._popupForm.reset();
+      
     });
   }
+
+  setButtonText(isFetching) {
+    const loadingText = "Saving...";
+    if (isFetching) {
+      this._submitBtn.textContent = loadingText;
+    } else {
+      this._submitBtn.textContent = "Save";
```

Comment:
Everything is almost fine, but:

* One remark hasn't been corrected https://github.com/marlondavid98/se_project_aroundtheus/pull/1#discussion_r1798952116 
* And this one https://github.com/marlondavid98/se_project_aroundtheus/pull/1#discussion_r1798952513 
* `avatarInformation` is not needed because you have already created an instance of `UserInfo`. Please, pass the 3rd selector iin line 77